### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fork79**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# testfork
\ No newline at end of file
+# testfork
+# second line
+# third line
```
Comment (asadmoosvi) [inline README.md]: change line to THIRD LINE

File: README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # testfork
 # second line
-# third line
+# THIRD LINE
```
Comment (asadmoosvi) [inline README.md]: Just add an exclamation point and then I'll merge.

**Comment by fork79** [comment]
On it.

**Comment by fork79** [comment]
I have added the  requested feature.

**Pull request merged**